<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton::hover{background:rgba(0,0,0,0.2);border-radius:5px;}</string>
   </property>
   <widget class="QLabel" name="lb_nota">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NOTA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="nama">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NAMA </string>
    </property>
   </widget>
   <widget class="QLabel" name="hp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NO. HP </string>
    </property>
   </widget>
   <widget class="QLabel" name="jumlah">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>JUMLAH </string>
    </property>
   </widget>
   <widget class="QLabel" name="berat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BERAT </string>
    </property>
   </widget>
   <widget class="QLabel" name="alamat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ALAMAT </string>
    </property>
   </widget>
   <widget class="QLabel" name="tglmasuk">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TGL MASUK </string>
    </property>
   </widget>
   <widget class="QLabel" name="paket">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>PAKET </string>
    </property>
   </widget>
   <widget class="QLabel" name="tglselesai">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TGL SELESAI </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kembali">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>KEMBALI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>-70</y>
      <width>751</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="judul">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Juice ITC</family>
      <pointsize>39</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nota Cuss Laundry</string>
    </property>
   </widget>
   <widget class="QLabel" name="nonota">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#9999ff;</string>
    </property>
    <property name="text">
     <string>NOTA:</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>20</width>
      <height>201</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="lineatas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="linekanan">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>20</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="linekiri">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>20</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="total">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TOTAL </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>436</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_nama">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_hp">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_alamat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_paket">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_berat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_jumlah">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_masuk">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_selesai">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_status">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#9999ff;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_total">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>436</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="Line" name="lineatas_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>lb_nota</zorder>
   <zorder>nama</zorder>
   <zorder>hp</zorder>
   <zorder>jumlah</zorder>
   <zorder>berat</zorder>
   <zorder>alamat</zorder>
   <zorder>tglmasuk</zorder>
   <zorder>paket</zorder>
   <zorder>tglselesai</zorder>
   <zorder>btn_kembali</zorder>
   <zorder>judul</zorder>
   <zorder>nonota</zorder>
   <zorder>line_6</zorder>
   <zorder>lineatas</zorder>
   <zorder>linekanan</zorder>
   <zorder>linekiri</zorder>
   <zorder>total</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>lb_nama</zorder>
   <zorder>lb_hp</zorder>
   <zorder>lb_alamat</zorder>
   <zorder>lb_paket</zorder>
   <zorder>lb_berat</zorder>
   <zorder>lb_jumlah</zorder>
   <zorder>lb_masuk</zorder>
   <zorder>lb_selesai</zorder>
   <zorder>lb_status</zorder>
   <zorder>lb_total</zorder>
   <zorder>lineatas_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
